frontend/src/pages/modelRegistry/screens/ModelRegistrySelector.tsx

```diff
@@ -82,46 +78,87 @@ const ModelRegistrySelector: React.FC<ModelRegistrySelectorProps> = ({
 
   const options = [
     <SelectGroup label="Select a model registry" key="all">
-      {modelRegistries.map((mr) => (
-        <SelectOption
-          id={mr.metadata.name}
-          key={mr.metadata.name}
-          value={getResourceNameFromK8sResource(mr)}
-          description={getMRSelectDescription(mr)}
-          isFavorite={favorites.includes(mr.metadata.name)}
-        >
-          {getDisplayNameFromK8sResource(mr)}
-        </SelectOption>
-      ))}
+      <SelectList>
```

Comment:
This select list extends past with viewport width:

<img width="500" alt="Screenshot 2024-08-02 at 11 13 27 AM" src="https://github.com/user-attachments/assets/439ce10b-d980-4424-bd1b-b36c6581b284">